mathics/builtin/base.py

```diff
@@ -859,6 +860,8 @@ def get_match_candidates(self, leaves, expression, attributes, evaluation, vars=
         return leaves
 
     def get_attributes(self, definitions):
+        if self.head is None:
```

Comment:
That is weird: why the head of a pattern would be None?

Reply:
https://github.com/Mathics3/mathics-core/blob/master/mathics/builtin/patterns.py#L1158 which goes back to th every first versions.

Reply:
@rocky I believe this deserves a comment.

Reply:
@TiagoCavalcante feel free to add one.

Reply:
Done.

Reply:
Done.